CONTRIBUTORS.md

```diff
@@ -63,6 +72,7 @@ The following individuals have contributed code to Galaxy:
 * Saskia Hiltemann <zazkia@gmail.com>
 * Rob Hooft <rob.hooft@nbic.nl>
 * Y. Hoogstrate <y.hoogstrate@erasmusmc.nl>
+* Hans-Rudolf Hotz <hrhotz@users.noreply.github.com>
```

Comment:
It's not, I used whatever the commits were signed with.  When I finish walking back through the missed releases, I'll @mention everyone and request tweaks.